Makefile

```diff
@@ -0,0 +1,60 @@
+#
+# Usage
+#    make <script from package.json>
+#
+# Runs all tasks from package.json in correct order and deps, so this can be parallelized.
+# (Good results observed up to -j4).
+#
+# By default, all output of successfull command is skipped, to enable it set VERBOSE env variable
+# like this:
+#
+#    VERBOSE=1 make
+
+tasks = \
+	tslint prettier \
+	test \
+	build-examples \
+	build-bundle \
+	build-tests \
+	test-browser-chrome test-browser-firefox \
+	typedoc
+
+default: $(tasks)
```

Comment:
is there any use cases of "default" rule? make without arguments runs first rule in Makefile by default

Reply:
No particular use case, other that readability.

> make without arguments runs first rule in Makefile
As you see, this rule is first for exactly this reason, you just need some name for it ... and `default` is actually good one

Reply:
Oh, I'm firstly took tasks declaration for the rule, didn't see the =